### PR TITLE
Removed extra prompt in appliance menu

### PR DESCRIPTION
**Pull request by yrudman**
Removed extra prompt in console menu : ```-- press enter/return to continue or q to stop--```.  

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1394054

There is new menu item was added to main appliance menu. This cause number of menu items reached limit after which pagination  of menu will start. 

**FIX:** increase setting for pagination

@miq-bot add-label bug

\cc @gtanzillo

### CODE REVIEW

File: lib/gems/pending/appliance_console.rb [hunk 1]
```diff
@@ -33,7 +33,8 @@
 I18n.backend.load_translations
 
 $terminal.wrap_at = 80
-$terminal.page_at = 25
+$terminal.page_at = 35
+
```
Comment (yrudman) [inline lib/gems/pending/appliance_console.rb]: right, done

**Comment by gtanzillo** [comment]
👍 Thanks @yrudman

**Comment by yrudman** [comment]
@miq_bot add-label euwe/yes

**Comment by bdunne** [comment]
@yrudman There is no euwe branch for ManageIQ/manageiq-gems-pending.  You'll need to create a PR against the euwe branch on ManageIQ/manageiq

**Comment by Fryguy** [comment]
> @yrudman There is no euwe branch for ManageIQ/manageiq-gems-pending. You'll need to create a PR against the euwe branch on ManageIQ/manageiq

@bdunne This is not correct.  The `euwe/yes` is a flag to @chessbyte so he knows to cherry-pick.  He can cherry-pick across repos, so having the euwe branch is not a requirement

**Comment by bdunne** [comment]
Okay, didn't know @chessbyte was going to cherry-pick across repos.

**Comment by yrudman** [comment]
blocker BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1396488

\cc @chessbyte

**Comment by chessbyte** [comment]
Euwe Backport details:
```sh
$ git log -1
commit d1abfcda0dacc327ae657c4650f808f0cd5da01a
Author: Brandon Dunne <brandondunne@hotmail.com>
Date:   Thu Nov 17 15:48:06 2016 -0500

    Merge pull request #16 from yrudman/fixed-console-extra-promptt
    
    Removed extra prompt in appliance menu
    (cherry picked from commit 2da80aad2f0b9838e97205598ee1c9d2c53b45d3)
    
    https://bugzilla.redhat.com/show_bug.cgi?id=1396488
```

**Pull request merged**